addon_service/migrations/0002_alter_externalstorageservice_int_addon_imp_and_more.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.7 on 2024-06-13 13:34
+
+import django.db.models.deletion
+from django.db import (
+    migrations,
+    models,
+)
+
+import addon_service.common.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("addon_service", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="externalstorageservice",
+            name="int_addon_imp",
+            field=models.IntegerField(
+                validators=[
+                    addon_service.common.validators.validate_storage_imp_number
+                ],
+                verbose_name="Addon implementation",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="externalstorageservice",
+            name="int_credentials_format",
+            field=models.IntegerField(
+                validators=[
+                    addon_service.common.validators.validate_credentials_format
+                ],
+                verbose_name="Credentials format",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="externalstorageservice",
+            name="int_service_type",
+            field=models.IntegerField(
+                default=1,
+                validators=[addon_service.common.validators.validate_service_type],
+                verbose_name="Service type",
+            ),
+        ),
+        migrations.AlterField(
+            model_name="externalstorageservice",
+            name="oauth2_client_config",
+            field=models.ForeignKey(
+                blank=True,
+                null=True,
+                on_delete=django.db.models.deletion.SET_NULL,
+                related_name="external_storage_services",
+                to="addon_service.oauth2clientconfig",
+            ),
+        ),
+    ]
```

Comment:
Looks like we made some model changes without `makemigrations`. @jwalz @aaxelb Do we want to add a new migration file or do we want to fold it into `0001_initial`?

Reply:
whoops, yeah i missed that

we are still updating `0001_initial` in place (`rm addon_service/migrations/000* && python manage.py makemigrations`), at least until we have external services set up on staging